### PR TITLE
Fix avatar fxa

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

File: kitsune/users/templatetags/jinja_helpers.py [hunk 1]
```diff
@@ -37,12 +38,14 @@ def profile_avatar(user, size=48):
     except (Profile.DoesNotExist, AttributeError):
         avatar = settings.STATIC_URL + settings.DEFAULT_AVATAR
     else:
-        if profile.is_fxa_migrated:
+        if (profile.fxa_avatar and
+                requests.head(profile.fxa_avatar).status_code == 200):
```
Comment (johngian) [inline kitsune/users/templatetags/jinja_helpers.py]: Why do you need this ?

Comment (johngian) [reply kitsune/users/templatetags/jinja_helpers.py]: This adds potential i/o overhead to all helper calls.

Comment (johngian) [reply kitsune/users/templatetags/jinja_helpers.py]: Also the default timeout is None which means that the request/response cycle relies on the availability of ``requests.head()``.

Comment (akatsoulas) [reply kitsune/users/templatetags/jinja_helpers.py]: The problem here is that while the user is logged in, the image can be removed from the management page of Firefox Accounts which has as a result a broken image everywhere where it's used.

**Comment by johngian** [comment]
After discussing it with @akatsoulas its a better solution to handle this on the client side because this has the potential of adding a lot of i/o overhead

**Pull request merged**